templates/_helpers.tpl

```diff
@@ -165,7 +177,7 @@ Reduce duplication of the st2.*.conf volume details
 {{- if index .Values "mongodb" "enabled" }}
 {{- $mongodb_port := (int (index .Values "mongodb" "service" "port")) }}
 - name: wait-for-db
-  image: busybox:1.28
+  image: {{ template "stackstorm-ha.imageRegistry" . }}/library/busybox:1.28
```

Comment:
Hmm. I don't use the private registry for these - so something like this might make sense.

Reply:
What if we added a value: `image.utilityImage` that defaults to `busybox:1.28`. That would make upgrading that image easier, and also allow updating it to include the repository.

Reply:
i can refactor to use that instead.